src/hooks.tsx

```diff
@@ -221,3 +222,52 @@ export const useIsInViewport = (
 
     return isInViewport;
 };
+
+export type BoundingRect = {
+    top: number;
+    right: number;
+    bottom: number;
+    left: number;
+    width: number;
+    height: number;
+    x: number;
+    y: number;
+};
+
+const getBoundingClientRect = (element: Element): BoundingRect => {
+    const {top, right, bottom, left, width, height, x, y} = element.getBoundingClientRect();
+    return {top, right, bottom, left, width, height, x, y};
+};
+
+export const useBoundingRect = (
+    ref: React.RefObject<Element>,
+    trackIfNotVisible?: boolean
```

Comment:
I don't see usages of this option, if not used, just remove it

Reply:
removed